contracts/ChipRegistry.sol

```diff
@@ -137,225 +124,224 @@ contract ChipRegistry is IChipRegistry, ClaimedPBT, Ownable {
-            tokenUri: _chipClaim.developerMerkleInfo.tokenUri,
-            tokenData: _encodeTokenData(_chipClaim.ersNode, _manufacturerValidation.enrollmentId)
-        });
-        // Mint a PBT this function fills out the ownership mapping, maps tokenId to chipId, fills out
-        // the chip table and increments the tokenIdCounter
-        uint256 tokenId = ClaimedPBT._mint(_chipClaim.owner, _chipId, chipInfo);
-
-        if (!projectInfo.claimsStarted) {
-            projectEnrollments[IProjectRegistrar(msg.sender)].claimsStarted = true;
+        if (!projectInfo.chipsAdded) {
+            projectEnrollments[projectRegistrar].chipsAdded = true;
         }
 
-        emit ChipClaimed(
+        emit ChipAdded(
             _chipId,
-            tokenId,
-            _chipClaim.owner,
-            _chipClaim.developerMerkleInfo.serviceId,
-            _chipClaim.ersNode,
+            address(projectRegistrar),
             _manufacturerValidation.enrollmentId,
-            _chipClaim.developerMerkleInfo.tokenUri
+            _chipOwner,
+            projectInfo.serviceId,
+            ersNode
         );
     }
 
+
     /**
-     * @notice Included for compliance with EIP-5791 standard but left unimplemented to ensure transfer policies can't be ignored.
+     * @dev ONLY Developer REGISTRAR: Remove project enrollment from ChipRegistry. This function is only callable by DeveloperRegistrars. This
+     * function will revert if the project is not enrolled or if the project has already added chips. This function will also remove the project
+     * subnode record in the ERS.
+     *
+     * @param _projectRegistrar          Address of the ProjectRegistrar contract
      */
-    function transferTokenWithChip(
-        bytes calldata /*signatureFromChip*/,
-        uint256 /*blockNumberUsedInSig*/,
-        bool /*useSafeTransfer*/
-    )
-        public
-        virtual
-        override(ClaimedPBT, IPBT)
-    {
-        revert("Not implemented");
+    function removeProjectEnrollment(IProjectRegistrar _projectRegistrar) external {
+        require(developerRegistry.isDeveloperRegistrar(msg.sender), "Must be Developer Registrar");
+        IDeveloperRegistrar developerRegistrar = IDeveloperRegistrar(msg.sender);
+
+        // Check that the project registrar is valid
+        require(address(_projectRegistrar) != address(0), "Invalid project registrar address");
+
+        // Verify that the project is enrolled
+        require(projectEnrollments[_projectRegistrar].creationTimestamp != 0, "Project not enrolled");
+
+        // Verify that the project is being removed by the correct developer registrar
+        require(projectEnrollments[_projectRegistrar].developerRegistrar == developerRegistrar, "Developer Registrar does not own project");
+
+        // Verify that the project has not added chips
+        require(projectEnrollments[_projectRegistrar].chipsAdded == false, "Cannot remove project with chips added");
```

Comment:
can change this to `require(!projectEnrollments[_projectRegistrar].chipsAdded, "Cannot remove project with chips added");

Reply:
Added in WIP commit.